slangpy/tests/device/test_device.py

```
@@ -147,18 +159,29 @@ def test_hot_reload_event(device_type: spy.DeviceType):
 
     # Setup a hook that increments a counter on hot reload.
     count = 0
+    unregistered_count = 0
 
-    def inc_count(x: spy.ShaderHotReloadEvent):
+    def inc_count(x: spy.ShaderHotReloadEvent) -> None:
```

Ruff (0.15.15)

[warning] Unused function argument: `x`

(ARG001)